setup-gems-cache/action.yml

```diff
@@ -0,0 +1,112 @@
+name: Setup Gems Cache
+description: Setup Gems Caching for workflow
+inputs:
+  ci_test_mode: 
+    description: Determine the composite action behavior on normal and ci test mode
```

Comment:
Can you be a bit more specific about what test mode does?

Reply:
It enables a step for cleaning up the CI machine.